### PR TITLE
{vis}[fosscuda/2019b] torchvision v0.5.0 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by crubb**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by crubb** [comment]
The error from the test runs:

```
found 2 variants of 'PyTorch' dependency in easyconfigs using '2019b' toolchain generation
225
* version: 1.3.1; versionsuffix: -Python-3.7.4 as dep for set(['PyTorch-Geometric-1.3.2-foss-2019b-Python-3.7.4.eb', 'torchvision-0.4.2-fosscuda-2019b-Python-3.7.4.eb', 'apex-20200325-fosscuda-2019b-Python-3.7.4.eb', 'Bonito-0.1.0-fosscuda-2019b-Python-3.7.4.eb', 'Bonito-0.1.0-foss-2019b-Python-3.7.4.eb'])
226
* version: 1.4.0; versionsuffix: -Python-3.7.4 as dep for set(['torchvision-0.5.0-fosscuda-2019b-Python-3.7.4.eb', 'torchtext-0.5.0-fosscuda-2019b-PyTorch-1.4.0--Python-3.7.4.eb', 'PyTorch-Geometric-1.4.2-foss-2019b-Python-3.7.4-PyTorch-1.4.0.eb'])
```

Unfortunately, according to https://github.com/pytorch/vision#installation,  torchvision 0.5.0 depends on PyTorch 1.4.

Should we wait for the 2020a toolchain?

**Comment by crubb** [comment]
Now PyTorch 1.5.0 has already been released ;)

**Comment by crubb** [comment]
Test report by @crubb
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
radbert - Linux ubuntu 18.04, Intel(R) Xeon(R) CPU E5-2687W v4 @ 3.00GHz, Python 2.7.17
See https://gist.github.com/0138be8d85975a523e6875b8d9438e58 for a full test report.

**Comment by crubb** [comment]
Test report by @crubb
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
radbert - Linux ubuntu 18.04, Intel(R) Xeon(R) CPU E5-2687W v4 @ 3.00GHz, Python 2.7.17
See https://gist.github.com/2314e406a105c2a152eb8a727ce9b987 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node154.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/bf1a88e019378f6a1ac748589d0fa8e9 for a full test report.

File: easybuild/easyconfigs/t/torchvision/torchvision-0.5.0-fosscuda-2019b-Python-3.7.4-PyTorch-1.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+use_pip = True
+download_dep_fail = True
+sanity_pip_check = True
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/torchvision'],
+}
+
```
Comment (lexming) [inline easybuild/easyconfigs/t/torchvision/torchvision-0.5.0-fosscuda-2019b-Python-3.7.4-PyTorch-1.4.0.eb:34]: These sanity checks are default for a `PythonPackage`, please remove them
```suggestion
```

File: easybuild/easyconfigs/t/torchvision/torchvision-0.5.0-fosscuda-2019b-Python-3.7.4-PyTorch-1.4.0.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+easyblock = 'PythonPackage'
+
+name = 'torchvision'
+version = '0.5.0'
+local_pytorch_ver = '1.4.0'
+versionsuffix = '-Python-%%(pyver)s-PyTorch-%s' % local_pytorch_ver
```
Comment (lexming) [inline easybuild/easyconfigs/t/torchvision/torchvision-0.5.0-fosscuda-2019b-Python-3.7.4-PyTorch-1.4.0.eb:6]: Minor suggestion: I think that we can drop the variable `local_pytorch_ver` as it is not used anywhere
```suggestion
versionsuffix = '-Python-%(pyver)s-PyTorch-1.4.0'
```

File: easybuild/easyconfigs/t/torchvision/torchvision-0.5.0-fosscuda-2019b-Python-3.7.4-PyTorch-1.4.0.eb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+sources = ['v%(version)s.tar.gz']
+checksums = ['eb9afc93df3d174d975ee0914057a9522f5272310b4d56c150b955c287a4d74d']
+
+builddependencies = [('CMake', '3.15.3')]
+
+local_pysuff = '-Python-%(pyver)s'
+dependencies = [
+    ('Python', '3.7.4'),
+    ('Pillow-SIMD', '6.0.x.post0'),
+    ('PyTorch', '1.4.0', local_pysuff),
```
Comment (lexming) [inline easybuild/easyconfigs/t/torchvision/torchvision-0.5.0-fosscuda-2019b-Python-3.7.4-PyTorch-1.4.0.eb:23]: Same here, I suggest to drop the variable `local_pysuff` as it not used anywhere else
```suggestion
dependencies = [
    ('Python', '3.7.4'),
    ('Pillow-SIMD', '6.0.x.post0'),
    ('PyTorch', '1.4.0', '-Python-%(pyver)s'),
```

**Comment by crubb** [comment]
Test report by @crubb
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
radbert - Linux ubuntu 18.04, Intel(R) Xeon(R) CPU E5-2687W v4 @ 3.00GHz, Python 2.7.17
See https://gist.github.com/4b4c16b4c947816e833d2e602823ac68 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node150.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/453ff7a346b95a8a8097fd473ecd0e56 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @crubb !

**Pull request merged**